main.go

```diff
@@ -204,8 +215,12 @@ func redirectIfNoCookie(handler http.Handler, r *http.Request, w http.ResponseWr
 			return
 		}
 		if !(config.AllowAll || admin == "true") {
-			writeHttpError(http.StatusForbidden)
-			return
+			allowed := config.AllowGroups.ContainsAny(groups)
```

Comment:
I think the mention of `AllowGroups` is self-explanatory, and we do not need the intermediate `allowed` variable, but if you find this easier to read then so be it.

Reply:
Yeah super minor, I think it is a tiny bit easier to read this way